cf_units/__init__.py

```diff
@@ -57,8 +57,13 @@
            'CALENDARS',
            'UT_NAMES',
            'UT_DEFINITION',
+           'UT_ASCII',
            'FLOAT32',
            'FLOAT64',
+           '_NO_UNIT',
+           '_NO_UNIT_STRING',
+           '_NO_UNIT_SYMBOL',
+           '_UNKNOWN_UNIT_SYMBOL',
```

Comment:
What is the benefit of adding these to the `__all__`? It's a bit unusual to add "private" objects to the `__all__` like this...